tests/Browser/FilterRoomsDuskTest.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+namespace Tests\Browser;
+
+use App\Models\Room;
+use App\Models\User;
+use Database\Seeders\EasyUserSeeder;
+use Database\Seeders\RolesAndPermissionsSeeder;
+use Database\Seeders\RoomSeeder;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Laravel\Dusk\Browser;
+use Tests\Browser\Components\DateTimePicker;
+use Tests\Browser\Pages\Bookings\Search;
+use Tests\DuskTestCase;
+
+class FilterRoomsDuskTest extends DuskTestCase
+{
+
+    use DatabaseMigrations;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+        $seeder = new EasyUserSeeder();
+        $seeder->run();
+        $seeder = new RolesAndPermissionsSeeder();
+        $seeder->run();
+        $seeder = new RoomSeeder();
+        $seeder->run();
+        User::first()->assignRole('super-admin');
+
+    }
+
+    /**
+     * Click on room additional filters,
+     * set number and checkbox filters and assert rooms
+     * go missing or stay in the search results
+     */
+    public function testRoomFilterByEventCriteria()
+    {
+        $this->browse(function (Browser $browser){
+
+            $admin = User::first();
+
+            if($admin === null) {
+
+                $admin = User::factory()->create();
+                $admin->assignRole('super-admin');
+            }
+
+            $browser->loginAs($admin);
+            $browser->visit('/bookings/search');
+
+            $browser
+                ->assertSourceHas('<title>CSU Booking Platform</title>');
+
+            $browser->press("@toggle-advanced-filters");
+            $browser->assertSourceHas('<h2>Capacity Standing</h2>');
+
+            $browser->assertSee('Art Nook');
+            $browser->type('#sofas', 1);
+            $browser->press("#filter-rooms");
+            $browser->pause(2000);
```

Comment:
```suggestion
            $browser->pause(250);
```

2 seconds is probably longer than needed

Reply:
Fixed